SomerenUI/EditDrinkForm.cs

```diff
@@ -29,6 +30,12 @@ public EditDrinkForm(Drink drink)
                 rdoFalse.Checked = false;
         }
 
+        protected void ShowMessageAndCloseForm(string message, Form form, string arg = null)
```

Comment:
I don't think this needs to be protected, since it's the child class and not parent class. Private should work, unless I'm overlooking something

Reply:
No, you're right. It's a copypaste mistake.